tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3646,26 +3648,47 @@ public function userSendsPropfindRequestToSpaceWithHeaders(string $user, string
 	 * @param string $spaceName
 	 * @param string|null $resource
 	 * @param array|null $headers
+	 * @param string|null $folderDepth
 	 *
 	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 *
 	 * @throws JsonException
 	 */
-	public function sendPropfindRequestToSpace(string $user, string $spaceName, ?string $resource = "", ?array $headers = []): ResponseInterface {
+	public function sendPropfindRequestToSpace(string $user, string $spaceName, ?string $resource = "", ?array $headers = [], ?string $folderDepth = "0"): ResponseInterface {
 		$this->setSpaceIDByName($user, $spaceName);
-		$properties = ['oc:permissions','oc:file-parent','oc:fileid','oc:share-types','oc:privatelink','d:resourcetype','oc:size','oc:name','d:getcontenttype','oc:tags','d:lockdiscovery','d:activelock'];
+		$properties = [
+			'oc:id',
+			'oc:fileid',
+			'oc:spaceid',
+			'oc:file-parent',
+			'oc:shareid',
+			'oc:name',
+			'd:displayname',
+			'd:getetag',
+			'oc:permissions',
+			'd:resourcetype',
+			'oc:size',
+			'd:getlastmodified',
+			'oc:tags',
+			'oc:favorite',
+			'oc:share-types',
+			'oc:privatelink',
+			'd:getcontenttype',
+			'd:lockdiscovery',
+			'd:activelock'
+		];
 		return  WebDavHelper::propfind(
 			$this->featureContext->getBaseUrl(),
 			$this->featureContext->getActualUsername($user),
 			$this->featureContext->getPasswordForUser($user),
 			$resource,
 			$properties,
 			$this->featureContext->getStepLineRef(),
-			"0",
+			$folderDepth,
```

Comment:
we can use default ocis value which is 1. then might not have to create new step
```suggestion
			"1",
```

Reply:
The step can be reusable for multiple depths.

Reply:
True, but we don't know when we might need it. But fine with me :+1: